rotkehlchen/api/rest.py

```diff
@@ -4262,21 +3415,18 @@ def add_manual_latest_price(
                 price=price,
             )
         except InputError as e:
-            return api_response(
-                result=wrap_in_fail_result(message=str(e)),
-                status_code=HTTPStatus.CONFLICT,
-            )
+            return wrap_in_fail_result(message=str(e), status_code=HTTPStatus.CONFLICT)
         Inquirer().remove_cache_prices_for_asset(pairs_to_invalidate)
 
-        return api_response(result=OK_RESULT)
+        return OK_RESULT
```

Comment:
Okay question. Why is this and the other manual latest price related endpoints changing from api response to dict?

Reply:
Hmm looks like the return type should have not changed. Probably I just automatically added the `@async_api_call()` decorator and because of that changed the return type.

In fact this function doesn't get called asynchronously, so probably the decorator should not be here and the return type should be `Response`.